src/categorical_algebra/ShapeDiagrams.jl

```diff
@@ -1,10 +1,20 @@
 """ Diagrams of a given shape.
 """
 module ShapeDiagrams
-export Span, Cospan, left, right, apex, base
+export AbstractFunctor, AbstractLaxitor, LaxMonoidalFunctor,
```

Comment:
`AbstractLaxitor` should be spelled `AbstractLaxator`.

Reply:
Fixed in latest commit.